### PR TITLE
.github: add kkleemola and pwnbus to codeowners and dependabot configs

### PR DESCRIPTION
**Pull request by g-k**
change dependabot update frequency to monthly

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -7,42 +7,42 @@ updates:
     timezone: UCT
```
Comment (g-k) [inline .github/dependabot.yml:7]: leaving docker updates as daily to get notified of point release with sec. fixes

**Comment by g-k** [comment]
@pwnbus should I add you as a codeowner too? It'd be good to have two team members familiar with the repo to review changes for each other.

**Comment by kkleemola** [comment]
+1 to someone/anyone besides just me.

**Comment by pwnbus** [comment]
@g-k yeah feel free to add me as codeowner as well

**Review by pwnbus — APPROVED** [review]

**Pull request merged**